query_optimizer/tests/execution_generator/Select.test

```diff
@@ -632,6 +632,48 @@ ORDER BY group_col1,
 +--------------------+--------------------+--------------------+
 ==
 
+
+CREATE TABLE dates (value DATETIME);
```

Comment:
I updated a test for `resolver`. For `logical_generator` and `physical_generator`, I think the purpose  is to check query plans. But `EXTRACT` is an expression which is not changed during logical planning and physical planning. So the resolver test should be sufficient.